### PR TITLE
[YUNIKORN-955] Move the web UI calls to the new REST API endpoints

### PR DESCRIPTION
**Pull request by akhilpb001**
### What is this PR for?
A few sentences describing the overall goals of the pull request's commits.
First time? Check out the contributing guide - http://yunikorn.apache.org/community/how_to_contribute   

### What type of PR is it?
* [ ] - Bug Fix
* [x] - Improvement
* [ ] - Feature
* [ ] - Documentation
* [ ] - Hot Fix
* [ ] - Refactoring

### Todos
* [ ] - Task

### What is the Jira issue?
* Open an issue on Jira https://issues.apache.org/jira/browse/YUNIKORN/
* Put link here, and add [YUNIKORN-*Jira number*] in PR title, eg. `[YUNIKORN-2] Gang scheduling interface parameters`

### How should this be tested?
Use docker image `akhilpb001/yunikorn-web:latestRestApi` to test this changes.

### Questions:
* [ ] - The licenses files need update.
* [x] - There is breaking changes for older versions.
* [ ] - It needs documentation.

### CODE REVIEW

File: src/app/services/scheduler/scheduler.service.ts [hunk 1]
```diff
@@ -167,68 +174,66 @@ export class SchedulerService {
...
-
-              result.push(nodeInfo);
-            });
-          }
+          data.forEach(node => {
+            const nodeInfo = new NodeInfo(
+              node['nodeID'],
+              node['hostName'],
+              node['rackName'],
+              node['partition'] || NOT_AVAILABLE,
```
[the remark this answers is not in the document]
Comment (akhilpb001) [reply src/app/services/scheduler/scheduler.service.ts:190]: This is the new nodes API response I got from Wilfred.
```
 {
      "nodeID": "node-1",
      "hostName": "",
      "rackName": "",
      "capacity": "[memory:1000 vcore:1000]",
      "allocated": "[memory:500 vcore:300]",
      "occupied": "[]",
      "available": "[memory:500 vcore:700]",
      "utilized": "[memory:50 vcore:30]",
      "allocations": [
        {
          "allocationKey": "alloc-1",
          "allocationTags": null,
          "uuid": "alloc-1-uuid",
          "resource": "[memory:500 vcore:300]",
          "priority": "0",
          "queueName": "root.default",
          "nodeId": "node-1",
          "applicationId": "app1",
          "partition": ""
        }
      ],
      "schedulable": true
    }
```

File: src/app/services/scheduler/scheduler.service.ts [hunk 2]
```diff
@@ -237,52 +242,61 @@ export class SchedulerService {
...
       const chilrenQs = [];
-      data.queues.forEach(queueData => {
+
+      data.children.forEach(queueData => {
         const childQueue = new QueueInfo();
-        childQueue.queueName = '' + queueData.queuename;
-        childQueue.state = queueData.status || 'RUNNING';
+
+        childQueue.queueName = queueData.queuename as string;
+        childQueue.status = queueData.status || 'n/a';
```
[the remark this answers is not in the document]
Comment (akhilpb001) [reply src/app/services/scheduler/scheduler.service.ts:252]: Fixed

File: src/app/components/queue-rack/queue-rack.component.ts [hunk 3]
```diff
@@ -61,21 +61,16 @@ export class QueueRackComponent implements OnInit {
     this.queueSelected.emit(queue);
   }
 
+  // FIXME: Implement using absolute usage value in new /{partition}/queues REST API.
+  // Currently absolute usage value is not available in /{partition}/queues REST API.
```
[the remark this answers is not in the document]
Comment (akhilpb001) [reply src/app/components/queue-rack/queue-rack.component.ts:65]: https://issues.apache.org/jira/browse/YUNIKORN-1128

File: src/app/services/scheduler/scheduler.service.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (akhilpb001) [reply src/app/services/scheduler/scheduler.service.ts:190]: @manirajv06 please add `partition` into the new the nodes response if it is not added or missing.

**Review by craigcondit — APPROVED** [review]
+1.

I've verified proper functionality with all the latest changes to core, and all looks good. Will merge shortly.

Thanks, Akhil for the great work!

**Pull request closed without merging**